### PR TITLE
Add charts/tables pagination

### PR DESCRIPTION
**Pull request by moonflare**
### What does it fix?
- Adds charts/tables pagination
- Reverses temperature chars axes
- Sort of fixes (https://github.com/code4romania/stam-acasa/issues/394 & https://github.com/code4romania/stam-acasa/issues/395)

⚠️  It waits for https://github.com/code4romania/taskforce-fe-components/pull/188 to fix the order of the data in the temperature chart.

-------

May-07-2020 00-14-18

-------

### CODE REVIEW

File: frontend/src/components/Account/common/ProfileHistory/ProfileHistory.js [hunk 1]
```diff
@@ -57,24 +63,117 @@ export const ProfileHistory = ({ data, family, isSelf }) => {
...
+          <div className="navigation">
+            {hasPrev(symptomsPage) && (
+              <div
+                className="button"
+                onClick={() => setSymptomsPage(symptomsPage - 1)}
+              >
+                <img
+                  src={ChevronImageLeft}
+                  className="navigation-chevron"
+                  alt="Pagina anterioara"
```
Comment (surdu) [inline frontend/src/components/Account/common/ProfileHistory/ProfileHistory.js]:
```suggestion
                  alt="Pagina anterioară"
```

File: frontend/src/components/Account/common/ProfileHistory/ProfileHistory.js [hunk 2]
```diff
@@ -57,24 +63,117 @@ export const ProfileHistory = ({ data, family, isSelf }) => {
...
+            )}
+            {hasNext(symptomsPage, history.symptoms.length) && (
+              <div
+                className="button right"
+                onClick={() => setSymptomsPage(symptomsPage + 1)}
+              >
+                <img
+                  src={ChevronImageRight}
+                  className="navigation-chevron"
+                  alt="Pagina urmatoare"
```
Comment (surdu) [inline frontend/src/components/Account/common/ProfileHistory/ProfileHistory.js]:
```suggestion
                  alt="Pagina următoare"
```

File: frontend/src/components/Account/common/ProfileHistory/ProfileHistory.js [hunk 3]
```diff
@@ -57,24 +63,117 @@ export const ProfileHistory = ({ data, family, isSelf }) => {
...
+          <div className="navigation">
+            {hasPrev(otherSymptomsPage) && (
+              <div
+                className="button"
+                onClick={() => setOtherSymptoms(otherSymptomsPage - 1)}
+              >
+                <img
+                  src={ChevronImageLeft}
+                  className="navigation-chevron"
+                  alt="Pagina anterioara"
```
Comment (surdu) [inline frontend/src/components/Account/common/ProfileHistory/ProfileHistory.js]:
```suggestion
                  alt="Pagina anterioară"
```

File: frontend/src/components/Account/common/ProfileHistory/ProfileHistory.js [hunk 4]
```diff
@@ -57,24 +63,117 @@ export const ProfileHistory = ({ data, family, isSelf }) => {
...
+            )}
+            {hasNext(otherSymptomsPage, history.otherSymptoms.length) && (
+              <div
+                className="button right"
+                onClick={() => setOtherSymptoms(otherSymptomsPage + 1)}
+              >
+                <img
+                  src={ChevronImageRight}
+                  className="navigation-chevron"
+                  alt="Pagina urmatoare"
```
Comment (surdu) [inline frontend/src/components/Account/common/ProfileHistory/ProfileHistory.js]:
```suggestion
                  alt="Pagina următoare"
```

File: frontend/src/components/Account/common/ProfileHistory/ProfileHistory.js [hunk 5]
```diff
@@ -57,24 +63,117 @@ export const ProfileHistory = ({ data, family, isSelf }) => {
...
+          <div className="navigation">
+            {hasPrev(outingsPage) && (
+              <div
+                className="button"
+                onClick={() => setOutings(outingsPage - 1)}
+              >
+                <img
+                  src={ChevronImageLeft}
+                  className="navigation-chevron"
+                  alt="Pagina anterioara"
```
Comment (surdu) [inline frontend/src/components/Account/common/ProfileHistory/ProfileHistory.js]:
```suggestion
                  alt="Pagina anterioară"
```

File: frontend/src/components/Account/common/ProfileHistory/ProfileHistory.js [hunk 6]
```diff
@@ -57,24 +63,117 @@ export const ProfileHistory = ({ data, family, isSelf }) => {
...
+            )}
+            {hasNext(outingsPage, history.outings.length) && (
+              <div
+                className="button right"
+                onClick={() => setOutings(outingsPage + 1)}
+              >
+                <img
+                  src={ChevronImageRight}
+                  className="navigation-chevron"
+                  alt="Pagina urmatoare"
```
Comment (surdu) [inline frontend/src/components/Account/common/ProfileHistory/ProfileHistory.js]:
```suggestion
                  alt="Pagina următoare"
```

**Review by surdu — DISMISSED** [review]

**Comment by aniri** [comment]
@moonflare version 1.0.25 of the components is live! :rocket:

**Comment by moonflare** [comment]
> @moonflare version 1.0.25 of the components is live! 🚀

Thanks @aniri! Will update the PR later tonight.

File: frontend/src/components/Account/common/ProfileHistory/ProfileHistory.js [hunk 7]
```diff
@@ -57,24 +63,117 @@ export const ProfileHistory = ({ data, family, isSelf }) => {
           <h2 className="header">Istoric Simptome</h2>
           <div className="profile-history-content">
             <TemperatureChart
-              results={history.temperature}
+              results={history.temperature.slice(
+                symptomsPage * PAGE_SIZE,
+                PAGE_SIZE * (symptomsPage + 1)
+              )}
               title="Monitorizare temperatura"
```
Comment (surdu) [inline frontend/src/components/Account/common/ProfileHistory/ProfileHistory.js]:
```suggestion
              title="Monitorizare temperatură"
```

**Review by surdu — DISMISSED** [review]
I found another one, but could be left for another diacritics pass. Though, @aniri would thank you 😆

**Review by surdu — APPROVED** [review]

**Review by RaduCStefanescu — APPROVED** [review]

**Pull request merged**